components/kyma-environment-broker/internal/provider/aws_provider.go

```diff
@@ -79,20 +88,114 @@ func ZoneForAWSRegion(region string) string {
 	if !found {
 		zones = "a"
 	}
+	rand.Seed(time.Now().UnixNano())
 	zone := string(zones[rand.Intn(len(zones))])
 	return fmt.Sprintf("%s%s", region, zone)
 }
 
+func MultipleZonesForAWSRegion(region string, zonesCount int) []string {
+	zones, found := awsZones[region]
+	if !found {
+		zones = "a"
+		zonesCount = 1
+	}
+	rand.Seed(time.Now().UnixNano())
```

Comment:
nit: similarly here, if we `rand.Seed` in init, we shouldn't need to call it here anymore